ompi/tools/mpirun/main.c

```diff
@@ -171,24 +124,14 @@ int main(int argc, char *argv[])
     // to Open MPI.
     setup_mca_prefixes();
 
-    /* calling mpirun (and now prterun) with a full path has a special
-     * meaning in terms of -prefix behavior, so copy that behavior
-     * into prterun */
-    if (opal_path_is_absolute(argv[0])) {
-        opal_argv_append_nosize(&prterun_args, full_prterun_path);
-    } else {
-        opal_argv_append_nosize(&prterun_args, "prterun");
+    
+    ret = prte_launch(argc, argv);
+    if (OMPI_SUCCESS != ret) {
+        opal_show_help("help-mpirun.txt", "prte-launch-failed", 1, strerror(errno));
```

Comment:
Amusingly, `help-mpirun.txt` doesn't even have a topic for the old name (`prterun-exec-failed`) -- possibly because this "fail to exec" error is so rare that it never actually happened in practice. 😄 

Can you add to this PR an edit in `help-mpirun.txt` (in this folder) a topic named `prte-launch-failed` that is a brief error message about how we failed to launch the user's application.